dockerfiles/Dockerfile.openvino

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: MIT
 #--------------------------------------------------------------------------
 
-ARG OPENVINO_VERSION=2023.0.0
```

Comment:
what about 2024.0 ?

Reply:
I think the team made the changes before 2024.0 was out. I see Linux OpenVINO CI failing too. I will run internal CI on 2024.0 to make sure that it is executed properly.